src/transformers/models/llama/configuration_llama.py

```diff
@@ -151,6 +151,11 @@ class LlamaConfig(PretrainedConfig):
         "layers.*.mlp.up_proj": "colwise",
         "layers.*.mlp.down_proj": "rowwise",
     }
+    base_model_pp_plan = {
```

Comment:
I think we can simplify a tad bit, we know we will always have:
0. input keys
1. output keys
so i would probably remove the dictionarry for that and just have two lists of lists!